plugin/trino-exchange-filesystem/src/main/java/io/trino/plugin/exchange/filesystem/FileSystemExchangeModule.java

```diff
@@ -53,7 +53,7 @@ protected void setup(Binder binder)
         if (scheme == null || scheme.equals("file")) {
             binder.bind(FileSystemExchangeStorage.class).to(LocalFileSystemExchangeStorage.class).in(Scopes.SINGLETON);
         }
-        else if (ImmutableSet.of("s3", "s3a", "s3n").contains(scheme)) {
+        else if (ImmutableSet.of("s3", "s3a", "s3n", "gs").contains(scheme)) {
```

Comment:
Unrelated to this PR, but we should limit S3 to just `s3` as the other schemes are legacy for Hive.